.github/scripts/validateActionsAndWorkflows.sh

```diff
@@ -42,12 +45,12 @@ done
 for ((i=0; i < ${#WORKFLOWS[@]}; i++)); do
   WORKFLOW=${WORKFLOWS[$i]}
 
-    # Skip linting e2e workflow due to bug here: https://github.com/SchemaStore/schemastore/issues/2579
-    if [[ "$WORKFLOW" == './workflows/e2ePerformanceTests.yml'
-          || "$WORKFLOW" == './workflows/testBuild.yml'
-          || "$WORKFLOW" == './workflows/deploy.yml' ]]; then
-      continue
-    fi
+  # Skip linting e2e workflow due to bug here: https://github.com/SchemaStore/schemastore/issues/2579
+  if [[ "$WORKFLOW" == './workflows/e2ePerformanceTests.yml'
+        || "$WORKFLOW" == './workflows/testBuild.yml'
+        || "$WORKFLOW" == './workflows/deploy.yml' ]]; then
+    continue
+  fi
```

Comment:
NAB:
* We typically use four spaces in bash vs two
* This could be shortened to `if [[ "$WORKFLOW" =~ ^(./workflows/e2ePerformanceTests.yml|./workflows/testBuild.yml|./workflows/deploy.yml)$ ]]; then`